Core/Security/ProxyFile.py

```diff
@@ -101,14 +105,14 @@ def multiProxyArgument( proxy = False ):
     if not proxy:
       proxyLoc = getProxyLocation()
       if not proxyLoc:
-        return S_ERROR( "Can't find proxy" )
+        return DError( DErrno.EPROXYFIND )
```

Comment:
Do we really need a special error for all the proxy related cases? Just a general EPROXY with a particular error message can be enough. I am a bit scared by the rapid proliferation of the error codes